src/Components/Form.js

```diff
@@ -0,0 +1,182 @@
+import axios from 'axios';
+import React,{useState} from 'react';
+import DatePicker from 'react-date-picker'
+import { useNavigate } from 'react-router-dom';
+import { FilePicker } from 'evergreen-ui';
+import "./form.css"
+const Form = () => {
+    const [firstname, setFirstname] = useState("");
+    const [lastname, setLastname] = useState("");
+    const [email, setEmail] = useState("");
+    const [mobile, setMobile] = useState("");
+    const [dob, setDob] = useState(new Date());
+    const [course,setCourse]=useState("Maths");
+    const [feedback,setFeedback]=useState("");
+    const [gender,setGender]=useState("");
+    const [agree,setAgree]=useState(false);
+    const [file,setFile]=useState("");
+    const navigate=useNavigate();
+    
+    const handleChangecourse =(e)=>{
+        setCourse(e.target.value)
+
+    }
+    const handleChangetext =(e)=>{
+        setFeedback(e.target.value)
+
+    }
+    const handleFirstName=(e)=>{
+        
+        
+        setFirstname(e.target.value)
+        
+
+    }
+    const handleLastName=(e)=>{
+        
+        setLastname(e.target.value)
+        
+
+    }
+    const handleEmail=(e)=>{
+        
+       
+        setEmail(e.target.value)
+    
+
+    }
+    const handleMobile=(e)=>{
+        
+        
+        setMobile(e.target.value)
+        
+
+    }
+    const handleAgree=(e)=>{
+        
+        
+            setAgree(!agree)
+        
+    }
+    const display =(e)=>{
+        e.preventDefault();
+       if(firstname === "")
+       {
+           alert("Please Enter a Valid first Name")
```

Comment:
Can you think of a better way to display validation messages instead of window alerts